ClearKeep/Scenes/ChatGroup/ChatGroupView.swift

```diff
@@ -0,0 +1,207 @@
+			.padding(.top, Constants.spacerTopView)
+
+			SearchTextField(searchText: $searchText,
+							inputStyle: $searchStyle,
+							inputIcon: AppTheme.shared.imageSet.searchIcon,
+							placeHolder: "GroupChat.Search.Placeholder".localized,
+							onEditingChanged: { isEditing in
+				if isEditing {
+					searchStyle = .normal
+				} else {
+					searchStyle = .highlighted
+				}
+			})
+				.padding(.top, Constants.paddingVertical)
+
+			HStack {
+				Text("Alissa Baker".localized)
+					.font(AppTheme.shared.fontSet.font(style: .body3))
+					.foregroundColor(foregroundTagUser)
+					.padding([.leading, .top, .bottom], Constants.paddingTagUser)
+				Button(action: {
+
+				}, label: {
+					Button {
+
+					} label: {
+						AppTheme.shared.imageSet.crossIcon
+							.foregroundColor(foregroundCrossIcon)
+							.padding(.trailing, Constants.paddingTagUser)
+					}
+				})
+			}
+			.background(backgroundTagUser)
+			.cornerRadius(Constants.cornerRadiusTagUser)
+			.frame(maxWidth: .infinity, alignment: .leading)
+			.frame(height: Constants.heightTagUser)
+
+			CheckBoxButtons(text: "GroupChat.User.Add.Title".localized, isChecked: $isShowingView)
+				.frame(maxWidth: .infinity, alignment: .leading)
+
+			if isShowingView {
+				CommonTextField(text: $searchText,
+								inputStyle: $searchStyle,
+								placeHolder: "GroupChat.User.Add.Placeholder".localized,
+								keyboardType: .default,
+								onEditingChanged: { isEditing in
+					if isEditing {
+						searchStyle = .default
+					} else {
+						searchStyle = .highlighted
+					}
+				})
+					.frame(maxHeight: .infinity, alignment: .top)
+				Spacer()
+			} else {
+				VStack {
+					HStack {
+						ZStack {
+							Circle()
+								.fill(backgroundGradientPrimary)
+								.frame(width: Constants.sizeImage, height: Constants.sizeImage)
+							AppTheme.shared.imageSet.userIcon
+						}
+						Text("Alissa Baker".localized)
+							.frame(minWidth: 0, maxWidth: .infinity, alignment: .leading)
+							.font(AppTheme.shared.fontSet.font(style: .input2))
+							.foregroundColor(foregroundTagUser)
+						CheckBoxButtons(text: "", isChecked: $isSelectedUser)
+					}
+					Spacer()
+				}
+			}
+
+			NavigationLink(
+				destination: EmptyView(),
+				label: {
+					Button(action: { },
+						   label: {
+						Text("GroupChat.Next".localized)
+					})
+						.frame(maxWidth: .infinity)
+						.frame(height: Constants.heightButton)
+						.font(AppTheme.shared.fontSet.font(style: .body3))
+						.background(backgroundGradientPrimary)
+						.foregroundColor(AppTheme.shared.colorSet.offWhite)
+						.cornerRadius(Constants.cornerRadiusButtonNext)
+						.padding(.horizontal, Constants.spacerTopView)
+				})
+				.padding(.bottom, Constants.paddingButtonNext)
+			Spacer()
+		}
+		.padding(.horizontal, Constants.paddingVertical)
+		.onReceive(inspection.notice) { inspection.visit(self, $0) }
+		.edgesIgnoringSafeArea(.all)
+		.navigationBarTitle("")
+		.navigationBarHidden(true)
```

Comment:
Make a view modifier and apply style navigation with empty title and hidden navigation bar